tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -2063,8 +2067,15 @@ def test_all_aggregates_in_query(self):
                 "max(transaction.duration)",
                 "avg(transaction.duration)",
                 "sum(transaction.duration)",
+                "stddev(transaction.duration)",
+                "var(transaction.duration)",
             ],
-            "query": "event.type:transaction min(transaction.duration):>1000 max(transaction.duration):>1000 avg(transaction.duration):>1000 sum(transaction.duration):>1000",
+            "query": (
+                "event.type:transaction min(transaction.duration):>1000 "
+                + "max(transaction.duration):>1000 avg(transaction.duration):>1000 "
+                + "sum(transaction.duration):>1000 stddev(transaction.duration):>=0.0 "
+                + "var(transaction.duration):>=0.0"
```

Comment:
Nit, this is very difficult to read and probably hard to edit in the future. Can we do something like

```python
" ".join([
  "event.type:transaction",
  "min(transaction.duration):>1000",
  ...
])
```